### ISSUE TITLE
userbouquet

### ISSUE THREAD

**Issue by navidyazdan** [DYefremov/DemonEditor#194]
Some service in userbouquet doesn't load in app

**Comment by DYefremov** [comment]
Sorry, but due to the complete missing of info (what bouquet, what services) it is difficult to comment on anything. It would be optimal if you send me a bouquet file indicating the services you are having a problem with.

**Comment by navidyazdan** [comment]
![Screenshot 2024-02-25 130513](https://github.com/DYefremov/DemonEditor/assets/30597524/87ed3893-04df-4f00-8940-d207f1f5bf41)
Line 3 and Line 9

**Comment by DYefremov** [comment]
it looks ok. Please post (as attachment) or send me your lamedb(5) file.

**Comment by DYefremov** [comment]
I had a little time to experiment...
Based on the above screenshot and service parameters, this should be a satellite in position 52.0E. Having checked several sites, for example [LyngSat](https://www.lyngsat.com/TurkmenAlem-MonacoSat.html) and others, I did not find any services with similar parameters. Perhaps you could indicate the names of the services? In addition, having downloaded several public sets of settings for motors (Vhannibal and the like), no such services were found also. Perhaps I missed something...

In general, without a set of your settings, it is difficult for me to confirm and check your case.

**Comment by navidyazdan** [comment]
[](url)
![photo_2024-03-01_21-00-57](https://github.com/DYefremov/DemonEditor/assets/30597524/14bd8649-133f-461f-8a9f-892b6149b199)
This is  bouquet from my Zero4k Gem Series HD and River HD is in bouquet

**Comment by navidyazdan** [comment]
![photo_2024-03-01_21-04-13](https://github.com/DYefremov/DemonEditor/assets/30597524/5cc9119e-5d79-4e45-aefb-35453c10df80)

This picture from demon editor there is no Gem Series HD and River HD

**Comment by navidyazdan** [comment]
![photo_2024-03-01_21-01-01](https://github.com/DYefremov/DemonEditor/assets/30597524/94dba659-d379-4ee5-97d9-da97dad91a95)
Total channel in Zero4K bouquet is 90

**Comment by navidyazdan** [comment]
![photo_2024-03-01_21-04-17](https://github.com/DYefremov/DemonEditor/assets/30597524/92b2fc9e-9e81-4f63-afae-250e8e30b300)
total channel in demon editor in 85

**Comment by navidyazdan** [comment]
[lamedb5.zip](https://github.com/DYefremov/DemonEditor/files/14464220/lamedb5.zip)

my lamedb(5) file

**Comment by navidyazdan** [comment]
I have not this problem with dreambox edit only with demon editor

**Comment by DYefremov** [comment]
Yes, this is an bug in reading _lamedb5_. Support for this format is still experimental.  I've added small fix.

![test](https://github.com/DYefremov/DemonEditor/assets/7511379/38e15c2a-c76e-4200-bc73-7a4c75c7b0ea)

If possible, check your settings for ver. 4 also. Thx.

**Comment by DYefremov** [comment]
I see you are using Windows. In your case, the easiest way is to wait for the update. Or, if you have the desire and time, you can try to [build and run](https://github.com/DYefremov/DemonEditor/blob/master/build/BUILD_WIN.md) it yourself. :)

**Comment by navidyazdan** [comment]
Thanks Dmitriy, for your help :-*

**Comment by DYefremov** [comment]
With the hope that everything is working we can close.
